crmsh/bootstrap.py

```diff
@@ -964,7 +964,9 @@ def ssh_copy_id_no_raise(local_user, remote_user, remote_node, shell: sh.LocalSh
         public_keys = ssh_key.fetch_public_key_file_list(None, local_user)
         sleep(5)    # bsc#1243141: sshd PerSourcePenalties
```

Comment:
NOTE:  5 seconds is a good practical choice. 4 seconds could still possible leads to the penalty during test.

for i in {1..50}; do ssh -o BatchMode=yes adm@server true; sleep 5; done